dev/integration_tests/android_embedding/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
```

Comment:
Are the two files above this supposed to be checked in? last-build.bin and gc.properties?

Reply:
I have no idea.  I don't know what those files are, but it would seem that they weren't covered by the standard .gitignore...